ignek_intranet/modules/employee/employee-web/src/main/resources/META-INF/resources/view.jsp

```diff
@@ -0,0 +1,179 @@
+<%@ taglib prefix="c" uri="http://java.sun.com/jsp/jstl/core"%>
+
+<liferay-ui:error embed="<%=false%>" key="error-for-existing-email"
+	message="Your email is already used, Try with another email." />
+
+<liferay-ui:success embed="<%=false%>" key="employee-created"
+	message="Employee Successfully Created" />
+
+<liferay-ui:error key="employee-not-created" embed="<%=false%>"
+	message="Employee not Created" />
+
+<liferay-ui:success embed="<%=false%>" key="employee-updated"
+	message="Employee Successfully Updated" />
+
+<liferay-ui:error embed="<%=false%>" key="employee-not-updated"
+	message="Employee not Updated" />
+
+<liferay-ui:success embed="<%=false%>" key="employee-deleted"
+	message="Employee Successfully Deleted" />
+
+<liferay-ui:error embed="<%=false%>" key="employee-not-deleted"
+	message="Employee not Deleted" />
+
+<portlet:actionURL var="addEmployeeActionURL">
+	<portlet:param name="mvcPath" value="/update-employee.jsp" />
+	<portlet:param name="empId" value="0" />
+</portlet:actionURL>
+
+<div class="portlet-background">
+	<div class="header-add">
+		<span class="page-heading">Employees</span> <span class="mb-5">
+			<c:if test="${hasRole == true}">
+			    <a href="<%=addEmployeeActionURL%>" class="btn text-white btn-default add-employee"> ADD NEW EMPLOYEE</a>
+			</c:if>
+		</span>
+	</div>
+
+	<liferay-portlet:renderURL varImpl="iteratorURL" />
+	<liferay-ui:search-container
+		total="<%=EmployeeLocalServiceUtil.getEmployeesCount()%>" delta="10"
+		emptyResultsMessage="no-employees-found"
+		iteratorURL="<%=iteratorURL%>" cssClass="table table-employee">
+		<%
+		int srNo = searchContainer.getStart() + 1;
+		%>
+		<liferay-ui:search-container-results>
+			<%
+			results = ListUtil.subList((List<Employee>) request.getAttribute("employeeList"), searchContainer.getStart(),
+					searchContainer.getEnd());
+			total = EmployeeLocalServiceUtil.getEmployeesCount();
+			pageContext.setAttribute("results", results);
+			pageContext.setAttribute("total", total);
+			%>
+		</liferay-ui:search-container-results>
+
+		<liferay-ui:search-container-row cssClass="table-row"
+			className="com.ignek.intranet.common.employee.model.Employee"
+			modelVar="emp" keyProperty="empId">
+
+			<portlet:actionURL var="updateEmployeeActionURL">
+				<portlet:param name="mvcPath" value="/save-employee.jsp" />
+				<portlet:param name="firstName" value="${emp.firstName}" />
+				<portlet:param name="lastName" value="${emp.lastName}" />
+				<portlet:param name="designation" value="${emp.designation}" />
+				<portlet:param name="phoneNumber" value="${emp.phoneNumber}" />
+				<portlet:param name="emailAddress" value="${emp.emailAddress}" />
+				<portlet:param name="addressLine1" value="${emp.addressLine1}" />
+				<portlet:param name="addressLine2" value="${emp.addressLine2}" />
+				<portlet:param name="city" value="${emp.city}" />
+				<portlet:param name="zipCode" value="${emp.zipCode}" />
+				<portlet:param name="empId" value="${emp.empId}" />
```

Comment:
Pass only ID here

You will need to create MVCRender class which will fetch the employee and redirect to the JSP

I have already explained this